docs/proposals/backup-restore-scenarios.md

```diff
@@ -45,13 +43,17 @@ This document illustrates real-world scenarios highlighting these challenges and
 
 ### Scenario 1: Creating Offline Snapshots for Air-Gapped Environments
 
-Dave, a security engineer at a FinTech company. To create a snapshot of the image and its referrers in an air-gapped environment, Dave needs to run the following flow:
+Dave is a security engineer at a FinTech company that operates in a highly regulated environment. As part of their infrastructure security policies, the production environment is fully air-gapped to reduce the attack surface and comply with regulatory standards.
+
+To deploy software to this air-gapped environment, Dave is responsible for preparing offline snapshots of all required container images and associated artifacts, including SBOMs, signatures, and attestations. These artifacts must be reviewed, signed, and bundled in a secure, verifiable manner before being transferred to production.
+
+The process typically involves:
 
-1. Packages the image and its referrers from an OCI image layout into a `.tar` for portability.
-2. Copies compressed files via secured channels to the air-gapped network.
-3. Restores all artifacts from a compressed file to an OCI registry in an air-gapped environment.
+1. Packaging an image and all its linked referrers from a remote registry into a single `.tar` archive or directory using OCI image layout format locally.
+2. Transferring the compressed snapshot over secure channels to the air-gapped network.
+3. Restoring the image and referrer rtifacts into an internal OCI-compliant registry for deployment.
 
-No unified snapshot solution available in `docker` or `oras`. Blobs duplicated across files and no assurance of artifact integrity and completeness causes problems to users. See [GitHub Issue #730](https://github.com/oras-project/oras/issues/730).
+Today, Dave has no native way to do this using `docker save/load`, `oras pull/push`, or other common tools. The lack of a unified snapshot solution means that artifacts must be copied manually or scripted with `oras copy`, often resulting in incomplete transfers (e.g., missing signatures or dependencies), blobs duplicated across files and no assurance of artifact integrity and completeness, difficulty validating artifact integrity upon restore. See an example GitHub issue [#730](https://github.com/oras-project/oras/issues/730) for details.
```

Comment:
> no assurance of artifact integrity and completeness, difficulty validating artifact integrity upon restore

Why?

Reply:
To validate artifact integrity, users need to check the supply chain artifacts such as signature, SBOM, etc. As mentioned above, the current solution (docker save/load or scripts with `oras copy`) may result in incomplete transfers (e.g., missing signatures or dependencies) which makes validation process hard.